### PR TITLE
Portuguese Translation

### PR DESCRIPTION
**Pull request by danielfbento**
First draft for Portuguese Translation of Contributor Covenant

 - add markdown
 - add text
 - add html

### CODE REVIEW

File: version/1/3/0/pt/code_of_conduct.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+Comportamentos abusivos, de assédio ou de outros tipos podem ser 
+comunicados contactando o administrador do projecto [INSIRA O ENDEREÇO 
+DE EMAIL]. Todas as queixas serão revistas e investigadas e 
+resultarão numa resposta necessária e apropriada à situação. 
+Os administradores são obrigados a manter a confidencialidade em relação
+ao elemento que reportou o incidente.
+
+Este Código de Conduta é adaptado do [Contributor Covenant](http://contributor-covenant.org), 
+versão 1.3.0, disponível em [http://contributor-covenant.org/version/1/3/0/](http://contributor-covenant.org/version/1/3/0/)
```
Comment (CoralineAda) [inline version/1/3/0/pt/code_of_conduct.md]: Can you change the URLs to reference the Portuguese versions? /1/3/0/pt/

Comment (danielfbento) [inline version/1/3/0/pt/code_of_conduct.md]: Fixed.

**Comment by CoralineAda** [comment]
@danielfbento Are you comfortable with the translation or do you want more time to work on this before merging?

**Comment by juanplopes** [comment]
Hi. This translation uses a luso-african orthography that can be easily understood, but it isn't generally accepted by Brazilian readers. I'm not sure if it is best to create a single text with a neutral orthography, or to add another Brazilian Portuguese translation. What do you think?

**Comment by CoralineAda** [comment]
I would think that the most widely-accepted form of the language would be best?

**Comment by juanplopes** [comment]
I think that's more of a subtle cultural problem, but I won't bother you with the details of the history of Brazil and Portugal. Let's just say that both versions of some words look really "weird" to both speakers.

While the Brazilian orthography has way more adepts (because there are more people here), the Luso-African is used in more countries. There is a "recent" official agreement, from 1990, trying to unify both orthographies, but it still allows both versions of some words (like "género" and "gênero", for gender). Also, the changes are more severe in the non-Brazilian side, resulting in low adoption outside Brazil.

**Comment by benlangfeld** [comment]
Responding to @CoralineAda's [request on Twitter](https://twitter.com/CoralineAda/status/669330813423480832), as a non-native but semi-fluent speaker of pt_BR, I can confirm that this is understandable, but as @juanplopes says, in some places "weird". I have requested the review of a linguistically-qualified non-technical native speaker of pt_BR, the result of which I will report here.

**Comment by danielfbento** [comment]
I have correct some portuguese typos that I found there. and removed trailing spaces. I don't know how to put the text assuming pt-BR and pt-PT at the same time. In Portugal we have always the two options. If anyone help =)

**Comment by danielfbento** [comment]
@benlangfeld any news?

**Comment by juanplopes** [comment]
I think it would be best to carry on with this pt-pt version (that looks good to me) and later propose any changes toward a more neutral text. This detour delayed this translation too much. I'm sorry for this.

**Comment by benlangfeld** [comment]
Apologies for the slow response. My reviewer agreed with my assessment and suggested a parallel pt_br. I will propose this tomorrow. In the meantime I would suggest merging this.

**Comment by danielfbento** [comment]
No problem. I asked a native speaker of pt_BR and he said to me that in some aspects it really hard to build a single valid text to pt_PT and pt_BR, for that he has the same opinion as you @benlangfeld.

**Comment by benlangfeld** [comment]
Again, apologies for the delay, but a pt_BR version is now available in #161.

**Pull request merged**